crates/pop-cli/src/commands/test/contract.rs

```diff
@@ -22,13 +28,30 @@ pub(crate) struct TestContractCommand {
 
 impl TestContractCommand {
 	/// Executes the command.
-	pub(crate) fn execute(self) -> anyhow::Result<&'static str> {
+	pub(crate) async fn execute(mut self) -> anyhow::Result<&'static str> {
 		clear_screen()?;
 		if self.features.is_some() && self.features.clone().unwrap().contains("e2e-tests") {
+			warning("--features e2e-tests is deprecated. Use --e2e instead.")?;
```

Comment:
This needs to be fixed. The `intro` should always be first, due to the various bar symbols used by cliclack.